add docs about chat_received webhook (#148)
team-plain/docs: api-reference/webhooks/thread-chat-received.mdx

---
title: 'Chat received'
---

> This event is fired when a chat message is sent by a customer in a thread.

Reviewer: ```suggestion
This event is fired when a chat message from a customer is received.
```
